examples/run/cpu/throughput_seq_example.cpp

```diff
@@ -0,0 +1,149 @@
+    traccc::spacepoint_formation sf(host_mr);
+    traccc::seeding_algorithm sa(host_mr);
+    traccc::track_params_estimation tp(host_mr);
+
+    traccc::demonstrator_input cells_event_vec;
+    cells_event_vec.reserve(i_cfg.loaded_events);
+
+    traccc::performance::timing_info elapsedTimes;
+
+    {
+
+        traccc::performance::timer t("File reading", elapsedTimes);
+
+        cells_event_vec = traccc::io::read(
+            i_cfg.loaded_events, i_cfg.input_directory, i_cfg.detector_file,
+            i_cfg.digitization_config_file, i_cfg.input_data_format, &host_mr);
+    }
+    // Seed random number generator
+    std::srand(std::time(0));
+
+    // Dummy count uses output of tp algorithm to ensure the compiler
+    // optimisations don't skip this step
+    std::size_t dummy_count = 0;
+
+    // Loop over events
+    for (int i = 0; i < i_cfg.processed_events; ++i) {
+
+        const int event = std::rand() % i_cfg.loaded_events;
+
+        {
+
+            traccc::performance::timer t("Events processing", elapsedTimes);
+
+            traccc::cell_container_types::host& cells_per_event =
+                cells_event_vec[event];
+
+            /*-------------------
+                Clusterization
+              -------------------*/
+
+            auto measurements_per_event = ca(cells_per_event);
+
+            /*------------------------
+                Spacepoint formation
+              ------------------------*/
+
+            auto spacepoints_per_event = sf(measurements_per_event);
+
+            /*-----------------------
+              Seeding algorithm
+              -----------------------*/
+
+            auto seeds = sa(spacepoints_per_event);
+
+            /*----------------------------
+              Track params estimation
+              ----------------------------*/
+
+            dummy_count += tp(spacepoints_per_event, seeds).size();
+        }
+    }
+    std::cout << "dummy_count = " << dummy_count << std::endl;
+
+    std::cout << "==> Statistics ... " << std::endl;
+    std::cout << "- Processed " << i_cfg.processed_events << " events from "
+              << i_cfg.loaded_events << " loaded events." << std::endl;
+
+    std::cout << "File reading throughput (ms/event) = "
+              << elapsedTimes.get_time("File reading").count() * 1.e-6 /
+                     (double)i_cfg.loaded_events
+              << std::endl;
+    std::cout << "Processing throughput   (ms/event) = "
+              << elapsedTimes.get_time("Events processing").count() * 1.e-6 /
+                     (double)i_cfg.processed_events
+              << std::endl;
+
+    return 0;
+}
+
+// The main routine
+//
+int main(int argc, char* argv[]) {
+    // Set up the program options
+    po::options_description desc("Allowed options");
+
+    // Add options
+    desc.add_options()("help,h", "Give some help with the program's options");
+    traccc::throughput_full_tracking_input_config batch_full_tracking_input_cfg(
```

Comment:
Since this variable is only used in these few lines, I would definitely make its name shorter. Plus, it still has "batch" in it. :stuck_out_tongue: